From a0179362231e2c1c4ebba7d5616da2a4677b1c4b Mon Sep 17 00:00:00 2001
From: Beniamino Galvani <bgalvani@redhat.com>
Date: Wed, 9 Sep 2020 10:59:12 +0200
Subject: [PATCH] device: fix wrongly considering ipv6.may-fail for ipv4

Fixes: 5e71f016057a ('device: merge stage3 and stage4 ip-config function for IPv4 and IPv6')

reason:fix wrongly considering ipv6.may-fail for ipv4

Conflict:NA
Reference:https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/commit/a0179362231e2c1c4ebba7d5616da2a4677b1c4b
---
 src/devices/nm-device.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/devices/nm-device.c b/src/devices/nm-device.c
index c325b40bc3..832116bd05 100644
--- a/src/devices/nm-device.c
+++ b/src/devices/nm-device.c
@@ -11748,7 +11748,7 @@ activate_stage4_ip_config_timeout_4 (NMDevice *self)
 	NMActStageReturn ret = NM_ACT_STAGE_RETURN_FAILURE;
 	NMDeviceStateReason failure_reason = NM_DEVICE_STATE_REASON_NONE;
 
-	ret = NM_DEVICE_GET_CLASS (self)->act_stage4_ip_config_timeout (self, AF_INET6, &failure_reason);
+	ret = NM_DEVICE_GET_CLASS (self)->act_stage4_ip_config_timeout (self, AF_INET, &failure_reason);
 	if (ret == NM_ACT_STAGE_RETURN_POSTPONE)
 		return;
 	else if (ret == NM_ACT_STAGE_RETURN_FAILURE) {
-- 
GitLab

